README.md

```diff
@@ -28,8 +28,102 @@ Please refer to [Heads-wiki](https://osresearch.net) for your Heads' documentati
 
 Building heads
 ===
+
+Under QubesOS?
+====
+* Setup nix persistent layer under QubesOS (Thanks @rapenne-s !)
+  * https://dataswamp.org/~solene/2023-05-15-qubes-os-install-nix.html
+* Install docker under QubesOS (imperfect old article of mine. Better somewhere?)
+  * https://gist.github.com/tlaurion/9113983bbdead492735c8438cd14d6cd
+
+Build docker from nix develop layer locally
+====
+
+If you do not use Nix on a daily basis:
+```
+# DANGER: remove /nix store and recreates a fresh one. Skip if you use Nix already:
+sudo rm -rf /nix/* || echo "cannot delete /nix" &&  sh <(curl -L https://nixos.org/nix/install) --no-daemon
+# Configure nix for local builds for nix-commands and flakes usage under nix which are considered experimental features
+mkdir -p ~/.config/nix
+echo 'experimental-features = nix-command flakes' >~/.config/nix/nix.conf
+# Source nix prior of anything else:
+. /home/user/.nix-profile/etc/profile.d/nix.sh
+```
+
+If you use Nix on a daily basis:
+```
+# Make sure your nix setup supports both nix-command and flakes experimental features:
+mkdir -p ~/.config/nix
+echo 'experimental-features = nix-command flakes' >> ~/.config/nix/nix.conf
+# Review ~/.config/nix/nix.conf for inconsistencies in your favorite editor (vim, vi, gedit etc)
+# Build nix developer local env with flakes locks to specified versions and exits just running "true" command:
+nix --print-build-logs --verbose develop --ignore-environment --command true
+# Build docker image with current develop created environment (this will take a while and create "linuxboot/heads:dev-env" local docker image:
+nix build .#dockerImage && docker load < result
+```
+
+Common steps to follow to build local doscker image from nix develop environment:
```

Comment:
Irrelevant anymore since used your suggestion for crunching README.md doc under [4b677eb](https://github.com/linuxboot/heads/pull/1661/commits/4b677eba795451f54e68363784b1743a61940178)